code/game/objects/machinery/mortar.dm

```diff
@@ -601,6 +603,13 @@
 	icon_state = "howitzer_ammo_wp"
 	ammo_type = /datum/ammo/mortar/smoke/howi/wp
 
+/obj/item/mortal_shell/howitzer/mustard
+	name = "\improper 150mm chemical X-50 'Mustard' artillery shell"
+	desc = "An 150mm artillery shell, loaded with an experimental gas known as X-50 which chokes, burns and lasts a long time. Known better as 'Mustard Gas' due to the bright yellow colorant it uses to denote its location to troops, whatever is hit by this will have their day, skin and future ruined, with a demand for a warcrime tribunal."
```

Comment:
```suggestion
	desc = "A 150mm artillery shell, loaded with an experimental X-50 toxic chemical gas which chokes, burns and lasts a long time. Known better as 'Mustard Gas' due to the bright yellow colorant it uses to denote its location to troops, whatever is hit by this will have their day, skin and future ruined, with a demand for a warcrime tribunal."
```